src/screens/CompareModels/CompareModels.js

```diff
@@ -59,17 +86,85 @@ export function CompareModels({ match, location }) {
     });
   }
 
+  const [sortType, setSortType] = useState(SORT_TYPES.ALPHABETICAL);
+  const [filterTypeIntervention, setFilterTypeIntervention] = useState(
+    DEFAULT_INTERVENTION_FILTER,
+  );
+
+  const sortFunctionMap = {
+    [SORT_TYPES.ALPHABETICAL]: sortAlphabetical,
+    [SORT_TYPES.OVERWHELMED]: sortByDateOverwhelmed,
+  };
+
   function refresh() {
     setLeftUrl(leftText);
     setRightUrl(rightText);
     setQueryParams(leftText, rightText);
     setRefreshing(true);
   }
 
+  function sortAlphabetical(a, b) {
+    return a < b ? -1 : 1;
+  }
+
+  function sortByDateOverwhelmed(a, b) {
+    const overwhelmedDifferenceA = getDifferenceInDateOverwhelmed(a);
+    const overwhelmedDifferenceB = getDifferenceInDateOverwhelmed(b);
+    return overwhelmedDifferenceA > overwhelmedDifferenceB ? -1 : 1;
+  }
+
+  function getDifferenceInDateOverwhelmed(stateAbbr) {
+    let overwhelmedLeft = getDateOverwhelmed(stateAbbr, leftModels);
+    let overwhelmedRight = getDateOverwhelmed(stateAbbr, rightModels);
+
+    if (overwhelmedLeft === overwhelmedRight) {
+      return 0;
+    } else if (overwhelmedLeft === null || overwhelmedRight === null) {
+      // if resources are never overwhelmed for only one of the models,
+      // return a large number to sort that model to the top
+      return 9999;
+    } else {
+      var dateOverwhelmedLeft = moment(overwhelmedLeft);
+      var dateOverwhelmedRight = moment(overwhelmedRight);
+      return Math.abs(
+        moment
+          .duration(dateOverwhelmedLeft.diff(dateOverwhelmedRight))
+          .asHours(),
+      );
+    }
+  }
+
+  function getDateOverwhelmed(stateAbbr, models) {
+    const currentIntervention = STATE_TO_INTERVENTION[stateAbbr];
+    const interventions = models[stateAbbr];
+    let interventionToModel = interventionToModelMap(interventions);
+    let model = interventionToModel[currentIntervention];
+
+    if (currentIntervention === INTERVENTIONS.SHELTER_IN_PLACE) {
+      model = interventionToModel[INTERVENTIONS.SOCIAL_DISTANCING];
+    }
```

Comment:
This logic is a bit surprising.  Is this how we do the "poor compliance" version of the intervention, I guess?  It's probably worth a comment and/or centralizing this somewhere so it doesn't get out of sync with the chart component...